keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -536,3 +537,57 @@ def _bpe_merge_and_update_cache(self, tokens):
             tokenized_words, axis=1, separator=" "
         )
         self.cache.insert(tokens, tokenized_words)
+
+    @classmethod
+    def from_preset(
+        cls,
+        preset,
+        **kwargs,
+    ):
+        """Instantiate a GPT-2 tokenizer from preset vocabulary and merge rules.
+
+        Args:
+            preset: string. Must be one of {{names}}.
+
+        Examples:
+        ```python
+        # Load a preset tokenizer.
+        tokenizer = keras_nlp.models.GPT2Tokenizer.from_preset(
+            "gpt2_base_en",
+        )
+        # Tokenize some input.
+        tokenizer("The quick brown fox tripped.")
+        # Detokenize some input.
+        tokenizer.detokenize([5, 6, 7, 8, 9])
+        ```
+        """
+
```

Comment:
After adding the `preset` property, check if empty as we do for backbone ([link](https://github.com/keras-team/keras-nlp/blob/99edccfe6388a2ea7f34f57fecb32c32cada33bc/keras_nlp/models/backbone.py#L66)). Same for the other two tokenizers.